pageserver/src/http/routes.rs

```diff
@@ -993,11 +993,24 @@ async fn tenant_status(
     check_permission(&request, Some(tenant_shard_id.tenant_id))?;
     let state = get_state(&request);
 
+    // In tests, sometimes we want to query the state of a tenant without auto-activating it if it's currently waiting.
+    let no_activate = parse_query_param(&request, "no_activate")?.unwrap_or(false);
```

Comment:
Even though this is a test only feature, would prefer to see positive booleans only, as in

```suggestion
    let no_activate = parse_query_param(&request, "activate")?.unwrap_or(true);
```

To avoid double-negatives.

Reply:
Hmmm... If it is test-only feature, shouldn't we also add a `cfg(feature = "testing")` in the code? I think this might work right away without warnings.

```rust
let activate = true;

#[cfg(feature = "testing")]
let activate = parse_query_param(&request, "activate").unwrap_or(activate);

if activate {
  tenant.wait_to_become_active(...).await.ok()
}
```

Reply:
Updated in 066688c2d